src/project-manager.ts

```diff
@@ -321,6 +428,11 @@ export class ProjectManager implements Disposable {
 		});
 	}
 
+	/**
+	 * Called when file was changed by client. Current implementation invalidates compiled version
+	 * @param fileName path to a file
+	 * @param text file's content
+	 */
 	didChange(fileName: string, text: string) {
 		this.localFs.didChange(fileName, text);
 		let version = this.versions.get(fileName) || 0;
```

Comment:
`versions` is documented as a Map from URI to version, but `fileName` is documented as a file path